README.md

```diff
@@ -163,9 +167,10 @@ Identity frontend is split between [new (profile-origin)](https://github.com/gua
 Responses:
 
     {
-      "adfree": true,
+      "adFree": true,
       "adblockMessage": false,
-      "userId": "123"
+      "userId": "123",
+      "membershipJoinDate": "2017-04-04"
     }
 
 ## Loading initial dataset - FIXME when would you want to do that?
```

Comment:
Is this section relevant? Can it just be removed?

Reply:
I think it (along with any code which loads a dataset) can probably just be removed.